config/locales/en.yml

```diff
@@ -360,11 +389,12 @@ en:
       personal_info_i: 'High Speed Two (HS2) Limited'
       personal_info_j: 'after the petition has been received by the Private Bill Office'
       personal_info_k: 'We will publish your petition on UK Parliament’s website. This will include your name and address.'
-      personal_info_l: 'The personal information you have provided may be kept in a database by both Private Bill Offices.'
+      personal_info_l: 'The personal information you have provided may be kept in a database by Private Bill Offices in both houses.'
+      personal_info_m: 'To find out more, read the <a href="https://www.parliament.uk/documents/foi/House-of-Lords-FOI/HL">House of Lords privacy notice</a>.'
```

Comment:
Privacy notice link does not work, not sure if this exists yet.

Reply:
Good spot @UsmanAfzal

Reply:
Thanks @UsmanAfzal, I've reported that to our content editor to review.

Reply:
That's fixed.